src/class-wpml-gutenberg-integration.php

```diff
@@ -152,6 +155,71 @@ private function update_block_translations( $blocks, $string_translations, $lang
 		return $blocks;
 	}
 
+	/**
+	 * @param array $block
+	 *
+	 * @return string
+	 */
+	private function render_block( $block ) {
+		$content = '';
+
+		if ( isset( $block['blockName'] ) ) {
+			$block_type = substr( $block['blockName'], 5 ); // strip core. eg core/paragraph
+
+			$block_attributes = '';
+			if ( $block['attrs'] ) {
+				$block_attributes = ' ' . json_encode( $block['attrs'] );
+			}
+			$content .= '<!-- wp:' . $block_type . $block_attributes . ' -->';
+
+			if ( isset ( $block['innerBlocks'] ) ) {
```

Comment:
*Note:* This part will be changed in the next task because it doesn't handle the Gutenberg columns block fully.